assets/js/script.js

```diff
@@ -19,21 +19,51 @@ var recipeHistoryEl = document.getElementById('recipe-history');
 //Event listener that runs search function on click
 
 var newSearch = document.getElementById('searchBtn');
-newSearch.addEventListener("click",fn1);
+newSearch.addEventListener("click", fn1);
 
 //Fetch list of recipe names from Spoonacular API based on ingredient inputs
 
+<<<<<<< HEAD
+function fn1(e) {
+
+
+    var ingred = document.getElementById('form1').value;
+    var ingred2 = document.getElementById('form2').value;
+    var ingred3 = document.getElementById('form3').value;
+    var allIngreds = ingred + ",+" + ingred2 + ",+" + ingred3;
+=======
 function fn1(e)
 {
    var ingred = document.getElementById('form1').value;
    var ingred2 = document.getElementById('form2').value;
    var ingred3 = document.getElementById('form3').value;
    var allIngreds = ingred + ",+" + ingred2 + ",+" + ingred3;
+>>>>>>> 7a79a776aa411c08a4edb5b7e50c1486719924d8
+
+    console.log(allIngreds);
+
+<<<<<<< HEAD
+    var newRecipe = 'https://api.spoonacular.com/recipes/findByIngredients?ingredients=' + allIngreds + '&number=10&apiKey=39791063581a4d96a908bb19745b3f64';
```

Comment:
Same with this, it might have been a merge conflict thing but it looks like this is a repeat of line 65?